bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/dialogs/KontaktErfassenDialog.java

```diff
@@ -403,7 +403,7 @@ private void createKontakt() {
 				}
 			}
 			if (LocalLockServiceHolder.get().acquireLock(newKontakt).isOk()) {
-				newKontakt.set(new String[] { "Strasse", "Plz", "Ort", "Telefon1", "Fax", "E-Mail" //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$ //$NON-NLS-5$ //$NON-NLS-6$
+				newKontakt.set(new String[] { "Strasse", "Plz", "Ort", "NatelNr", "Fax", "E-Mail" //$NON-NLS-1$//$NON-NLS-2$//$NON-NLS-3$//$NON-NLS-4$ //$NON-NLS-5$ //$NON-NLS-6$
 				}, new String[] { ret[4], ret[5], ret[6], ret[7], tFax.getText(), tEmail.getText() });
```

Comment:
Wird da jetzt einfach die MobileNr in Tel1 reingeschrieben? Ansonsten müsstest du in Zeile 407 ja auch das Ziel ändern!